app/views/articles/_bottom_content.html.erb

```diff
@@ -7,17 +7,8 @@
     <a href="<%= article.path %>" data-preload-image="<%= cloud_cover_url(article.main_image) %>">
       <div class="single-other-article">
         <div class="picture">
-          <%= cl_image_tag(article.cached_user.profile_image_url,
-                           type: "fetch",
-                           crop: "imagga_scale",
-                           width: 100,
-                           height: 100,
-                           quality: "auto",
-                           flags: "progressive",
-                           fetch_format: "auto",
-                           loading: "lazy",
-                           sign_url: true,
-                           alt: "#{article.cached_user.username} profile image") %>
+          <% image_url = Images::Optimizer.call(article.cached_user.profile_image_url, crop: "imagga_scale", width: 100, height: 100) %>
```

Comment:
**suggestion (non-blocking):** I wonder if it could make sense to wrap this up in a combined helper, something like

```ruby
def optimized_image_tag(url, optimizer_options: {}, image_options: {})
  image_option[:width] ||= optimizer_options[:width]
  image_option[:height] ||= optimizer_options[:height]
  image_url = Images::Optimizer.call(url, optimizer_options)

  image_tag(image_url, image_options)
end
```

Which would be called like this:

```ruby
<%= optimized_image_url(article.cached_user.profile_image_url, 
                        optimizer_options: { crop: "imagga_scale", width: 100, height: 100 },
                        image_options: { loading: "lazy", alt: "#{article.cached_user.username} profile image" })
%>
```

(This is untested, just meant to illustrate the idea)

Reply:
🤯  Thank you so much for this suggestion! 

This is exactly what I wanted. I was making things harder for myself because I thought we shouldn't be using more than than one `option` hash as args. Updating now.